op-challenger/scripts/visualize.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ $# -eq 0 ]
+  then
+    echo "Missing Fault Dispute Game address argument"
+fi
+
+DISPUTE_GAME_PROXY=$(jq .DisputeGameFactoryProxy .devnet/addresses.json)
+DISPUTE_GAME_PROXY=$(echo $DISPUTE_GAME_PROXY | tr -d '"')
+
+echo "----------------------------------------------------------------"
+echo " Dispute Game Factory at $DISPUTE_GAME_PROXY"
+echo "----------------------------------------------------------------"
+
+FAULT_GAME_ADDRESS=$1
+
+dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+cd "$dir"
+cd ../packages/contracts-bedrock
+
+forge script scripts/FaultDisputeGameViz.s.sol --sig "remote(address)" $FAULT_GAME_ADDRESS --fork-url http://localhost:8545
+mv dispute_game.svg "$dir"
```

Comment:
Maybe we can link to the more modern visualizer in this script which hooks into the local devnet? URL schema is:
```
https://dispute.clab.by/game?addr=<address>
```
Optional, though, as this viz script will work for the Cannon game with a much lower depth. Could switch depending on the game type, maybe.

Reply:
We probably shouldn't embed links to external websites in the monorepo. Plus the hosted version won't work without making some pretty insecure changes to allow insecure requests from the https hosted site.